src/endpoint/egress.h

```diff
@@ -24,17 +26,49 @@
 
 namespace endpoint {
 
-
-
 class Egress : public Endpoint {
 private:
+    /**
+     * Method used to manage UDP packets received from the chain.
```

Comment:
_Used to_ [refers to hold habits in the past](https://dictionary.cambridge.org/it/grammatica/grammatica-britannico/past/used-to), it should not be used here
```suggestion
     * It manages incoming UDP packets that it has received from the chain
```

Reply:
done